install.sh

```diff
@@ -1,41 +1,58 @@
 #!/bin/sh
 
-# Get NordVPN Repo Setup (https://nordvpn.com/download/linux/)
-echo Getting the NordVPN Repo Setup
-wget https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/nordvpn-release_1.0.0_all.deb
-
-# Install downloaded package
-echo Installing the NordVPN Repo Setup
-sudo apt-get install nordvpn-release_1.0.0_all.deb
-rm nordvpn-release_1.0.0_all.deb
-
-# Update packages
-echo Updating packages
-sudo apt-get update
-
-# Install NordVPN
-echo Installing NordVPN
-sudo apt-get install nordvpn
-
-# Prompting user to login
-echo Logging into NordVPN
-nordvpn login
-
-# Install gir1.2-appindicator
-echo Installing AppIndicator
-sudo apt-get install gir1.2-appindicator
-
-# Installing indicator in opt directory
-echo Installing Ubuntu NordVPN Indicator
-sudo mkdir /opt/ubuntu-nordvpn-indicator/
-sudo cp code/nordvpn_* /opt/ubuntu-nordvpn-indicator/
-
-# Installing autostart desktop file
-echo Making sure the indicator starts at boot using autostart
-cp ubuntu-nordvpn-indicator.desktop ~/.config/autostart/
+install_client()
+{
+    # Get NordVPN Repo Setup (https://nordvpn.com/download/linux/)
+    echo Getting the NordVPN Repo Setup
+    wget https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/nordvpn-release_1.0.0_all.deb
+
+    # Install downloaded package
+    echo Installing the NordVPN Repo Setup
+    sudo apt-get install -y nordvpn-release_1.0.0_all.deb
+    rm nordvpn-release_1.0.0_all.deb
+
+    # Update packages
+    echo Updating packages
+    sudo apt-get update
+
+    # Install NordVPN
+    echo Installing NordVPN
+    sudo apt-get install -y nordvpn
+
+    # Prompting user to login
+    echo Logging into NordVPN
+    nordvpn login
+}
+
+install_deps()
+{
+    # Install gir1.2-appindicator
+    echo Installing AppIndicator
+    sudo apt-get install -y gir1.2-appindicator
+}
+
+install_indicator()
+{
+    # Installing indicator in opt directory
+    echo Installing Ubuntu NordVPN Indicator
+    sudo mkdir -p /opt/ubuntu-nordvpn-indicator/
+    sudo cp code/nordvpn_* /opt/ubuntu-nordvpn-indicator/
+
+    # Installing autostart desktop file
+    echo Making sure the indicator starts at boot using autostart
+    cp ubuntu-nordvpn-indicator.desktop ~/.config/autostart/
```

Comment:
If autostart does not exist (i.e.: Ubuntu 18.04), this willl crash
```suggestion
    mkdir -p ~/.config/autostart/
    cp ubuntu-nordvpn-indicator.desktop ~/.config/autostart/
```